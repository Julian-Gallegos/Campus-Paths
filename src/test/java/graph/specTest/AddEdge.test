# Tests that an edge can be created between a parent node and a child node.

# Create an empty graph
CreateGraph graph1

# Add nodes to a graph
AddNode graph1 node1
AddNode graph1 node2

# Add an edge from node1 to node 2
AddEdge graph1 node1 node2 edge1