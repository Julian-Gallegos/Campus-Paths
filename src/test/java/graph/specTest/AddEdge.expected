# Tests that an edge can be created between a parent node and a child node.

# Create an empty graph
created graph graph1

# Add nodes to a graph
added node node1 to graph1
added node node2 to graph1

# Add an edge from node1 to node 2
added edge edge1 from node1 to node2 in graph1